### ISSUE TITLE
Inaccessible as a modal

### ISSUE THREAD

**Issue by djmtype** [rqrauhvmra/Tobii#28]
When using tobii as a modal, the modal contents are inaccessible via keyboard. They can't be focused on or read using the controls. Tested with Voiceover, Chrome on MacOS. 

```
var tobii = new Tobii({
  swipeClose: false,
  draggable: false,
})
```

**Comment by viliusle** [comment]
Feel free to reopen, but it should be fixed on next version. Already fixed on develop branch.
